Backend/Dev/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using ManagerLayer;
+using ManagerLayer.UserManagement;
+using System;
+<<<<<<< HEAD
+using DataAccessLayer.Database;
+using ServiceLayer.Services;
+using DataAccessLayer.Models;
+using UnitTesting;
+=======
+>>>>>>> 9de364cebb612877c7f3127cd8a6d9c1f7839428
+
+namespace Program
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+<<<<<<< HEAD
```

Comment:
Is this a merge conflict that has not been resolved?

Reply:
Good looks alfredo!